### ISSUE TITLE
LDO instable - input capacitance to low

### ISSUE THREAD

**Issue by mkiepiela** [sinara-hw/Kasli-SOC#68]
Input capacitance at IC35 is too low (only C125 = 1 uF). It causes voltage instability both, at the output (1V0) and input (1V8) side. As an effect MGTs transceivers may not work correctly.

<img src="https://user-images.githubusercontent.com/25741678/181057460-89f883e0-ee30-476f-a3d1-40cad9dffe8f.png" width="400">

<img src="https://user-images.githubusercontent.com/25741678/181058285-fca9c521-5a73-426b-bbc4-d0a2c56c60dd.png" width="400">

After adding 2.2 uF capacitor in parallel to C125, supply voltage is correct and MGTs work well.
Basically, all LDOs should be reviewed, as there are more LDOs with 1 uF input capacitance. Probably some of them also should be increased.

**Comment by gkasprow** [comment]
Good catch.
Moreover, take into account C=f(V) effect in ceramic capacitors. Real capacitance is often twice lower than rated.

**Comment by nkrackow** [comment]
So I think the other affected caps are C118, C113 and C130? Or are there more at other places that I might have missed?

**Comment by mkiepiela** [comment]
Yes, it looks like there are only these 3 caps more as you pointed above. However I looked into datasheet of LDO and it suggests using 1 uF input cap in combination with low line impedance.

<img src="https://user-images.githubusercontent.com/25741678/182571434-3cf0a08d-a3cf-4922-9607-e67bba2aff46.png" width="600">

So far we encountered the problem only with 1V0 LDO when MGT transceivers were active (high current consumption from 1V0 domain). Other voltages were stable and did not need additional capacitance.
Moreover, there is the least dropout value on this particular LDO (1V8 - 1V0 = 800 mV) what increases the risk of instability.

The next one with low dropout value (1.35 V) is IC33 (1V35 VCCDDR), however we did not observe instability there. But for now we were able to check it only on one board.

Summarizing, I think that adding capacitor only at 1V0 LDO seems to be a good enough solution, unless someone observe similar problem on any other LDO.

**Comment by nkrackow** [comment]
Ok, thanks!

**Comment by filipswit** [comment]
Added 10uF at inputs for all 4 LDOs
